css/style.css

```diff
@@ -0,0 +1,34 @@
+header {
+    position: fixed;
+    width: 100%;
+  }
+
+body{

```

Comment:
- [ ] I think you should add a white space after `body` to keep CSS best practices 🤓